src/routes/index.ts

```diff
@@ -77,43 +63,62 @@ router.get('/:type/all', async (ctx) => {
   }
 });
 
-router.get('/:type/:id', async (ctx) => {
-  try {
-    const { lang } = ctx.query;
-    const { type, id } = ctx.params;
-
-    ctx.body = await getEntity(type, id, lang as string);
-  } catch (e) {
-    ctx.status = 404;
-    ctx.body = { error: e.message };
-  }
-});
-
-router.get('/:type/:id/list', async (ctx) => {
-  const { type, id } = ctx.params;
-
+router.get('/:type*/list', async (ctx) => {
+  let { type } = ctx.params;
+  const id = type.substring(type.lastIndexOf("/") + 1);
+  type = type.substring(0, type.lastIndexOf("/"));
   try {
+    if(await containsFolders(ctx.params.type))
+      throw new Error(`No images for ${ctx.params.type}/lista exist`);
```

Comment:
what does `lista` mean here? should that be there?

Reply:
Some leftover from debugging.
Already fixed.

Reply:
Just a typo